src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -142,6 +143,7 @@ public StreamingResponseBody streamEvents(
                         streamLimit, batchTimeout, streamTimeout, streamKeepAliveLimit);
 
                 response.setStatus(HttpStatus.OK.value());
+                response.addHeader(HttpHeaders.CONNECTION, "close");
```

Comment:
For now it will have no effect. But imho it's better to say that this connection must be closed instead of reusing it. We do not know how ELB is implemented, so maybe it will try to reuse connection (some day), but we expect that streaming will be ended only when connection is closed. That's why I added it.